src/main/resources/application.properties

```diff
@@ -1 +1,8 @@
+spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
+spring.datasource.url=jdbc:mysql://localhost:3306/spring-boot
+spring.datasource.username=root
+spring.datasource.password=Root-159363
```

Comment:
I hope it is not your password that pass for a lot of your accounts, in other order delete it form here.